### PR TITLE
Soumyap/remork rename

### PR DESCRIPTION
**Pull request by soumyapani**

### CODE REVIEW

File: tools/terraform-doc.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+for file in "$@"; do
+	paths[index]=$(dirname "$file")
+	((index += 1))
+done
+
+uniq_paths=$(echo "${paths[@]}" | tr ' ' '\n' | sort -u)
+
+for path in $uniq_paths; do
+	terraform-docs markdown --config .tfdocs-markdown.yaml "${path}"
+done
```
Comment (stevenBorisko) [inline tools/terraform-doc.sh:27]: nit: could this be simplified with some pipes? maybe something like:
```bash
for file in "${@}"; do dirname "${file}"; done \
| sort -u \
| while read dir; do terraform-docs markdown --config .tfdocs-markdown.yaml "${dir}"; done
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    - repo: https://github.com/antonbabenko/pre-commit-terraform
+      rev: v1.77.1
+      hooks:
+        - id: terraform_fmt
+          always_run: true
+    - repo: local
+      hooks:
+      - id: terraform-readme
+        name: terraform-readme
+        entry: tools/terraform-doc.sh
```
Comment (stevenBorisko) [inline .pre-commit-config.yaml:11]: nit: inconsistent tabbing in `hooks`

**Review by stevenBorisko — APPROVED** [review]
lgtm -- non-blocking consistency/readability comments

File: tools/terraform-doc.sh (hunk 1 above)
Comment (soumyapani) [reply tools/terraform-doc.sh]: done.

**Pull request merged**